src/popup/router/components/AssetSelector.vue

```diff
@@ -0,0 +1,96 @@
+<template>
```

Comment:
Please skip this file while reviewing as it will be totally changed by @pstachel .